### DISCUSSION TITLE
hacktoberfest

### DISCUSSION THREAD

**Discussion by clxmstaab** [deployphp/deployer#2217] [General]
would you mind adding this project to the [Hacktoberfest ](https://hacktoberfest.digitalocean.com/)?

only thing you need to do is adding a repo-topic "hacktoberfest"

see for a example: 
![image](https://user-images.githubusercontent.com/47448731/97010352-91ebab80-1545-11eb-81ca-a4a032253016.png)

**Comment by clxmstaab** [comment]
background: would be great to get the PRs counted towards the Hacktoberfest goal

**Comment by antonmedv** [comment]
Yes, why not.
